demos/dialog/index.js

```diff
@@ -1,3 +1,5 @@
+import {LitElement, html} from 'lit-element/lit-element.js';
```

Comment:
I would avoid using LitElement in these demos. We want to show off pure JS that can copy/pasted anywhere, and using lit defeats that purpose.

Reply:
totally! i was still in lit-html mode from some pair coding with Dan.
I can do pure JS! i like pure JS.